sdk/ml/azure-ai-ml/azure/ai/ml/entities/__init__.py

```diff
@@ -65,6 +65,7 @@
     ManagedOnlineDeployment,
     OnlineDeployment,
 )
+from ._deployment.data_collector import DataCollector
```

Comment:
is `DataCollector` supposed to be `experimental`?

Reply:
Yes, I just mark it as experimental